### ISSUE TITLE
Accessibility fix hit list

### ISSUE THREAD

**Issue by GBKS** [BitcoinDesign/Meta#516]
This is a tracking issue for accessibility issues with open-source, self-custodial bitcoin wallets that we identified and want to help fix. We unearthed lots of issues in the research for the [accessibility page](https://bitcoin.design/guide/designing-products/accessibility/) in the guide, and we're organizing an [event](https://github.com/BitcoinDesign/Meta/issues/492) for the Global Accessibility Awareness Day. Let's use this issue to keep track of practical improvements we're making (or facilitating).

Issues below are by no means complete. This is just what we found in spot testing mid 2022. Any help with these is highly appreciated.

## Blue Wallet
Written in React Native.

- [ ] [Improve accessibility of floating buttons (scan, send, receive)](https://github.com/BlueWallet/BlueWallet/issues/5389)
- [ ] [Make QR code on receive screen accessible](https://github.com/BlueWallet/BlueWallet/issues/5388)
- [x] [Add missing accessibility labels](https://github.com/BlueWallet/BlueWallet/pull/5390)

## Muun

- [ ] [Accessibility/labelling issues](https://github.com/muun/falcon/issues/23)

## Blockstream Green

- [ ] [Accessibility test results & recommendations](https://github.com/Blockstream/green_ios/issues/43)

## Phoenix
[Test results](https://www.figma.com/file/vWxA4SRJq7RdXWiKrB032y/Bitcoin-app-accessibility-tests?node-id=106%3A3453&t=FFaJxIGmxna1hwKT-1) (Figma) from June 2022. Not filed as issues yet.

## Breez
[Test results](https://www.figma.com/file/vWxA4SRJq7RdXWiKrB032y/Bitcoin-app-accessibility-tests?node-id=52%3A1816&t=FFaJxIGmxna1hwKT-1) (Figma) from May 2022. Not filed as issues yet.

**Comment by mouxdesign** [comment]
## Blue Wallet
Written in React Native.

- [ ]  https://github.com/BlueWallet/BlueWallet/issues/5389
- [ ]  https://github.com/BlueWallet/BlueWallet/issues/5388
- [ ]  https://github.com/BlueWallet/BlueWallet/pull/5390

## Muun

- [ ]  https://github.com/muun/falcon/issues/23

## Blockstream Green

- [ ]  https://github.com/Blockstream/green_ios/issues/43

## Zeus

- [ ] https://github.com/ZeusLN/zeus/issues/1442

[Test results](https://www.figma.com/file/vWxA4SRJq7RdXWiKrB032y/Bitcoin-app-accessibility-tests?node-id=123%3A2502&t=FFaJxIGmxna1hwKT-1) (Figma) from July 2022. Not filed as issues yet.

## Phoenix
- [Test results](https://www.figma.com/file/vWxA4SRJq7RdXWiKrB032y/Bitcoin-app-accessibility-tests?node-id=106%3A3453&t=FFaJxIGmxna1hwKT-1) (Figma) from June 2022. Not filed as issues yet.

## Breez
- [Test results](https://www.figma.com/file/vWxA4SRJq7RdXWiKrB032y/Bitcoin-app-accessibility-tests?node-id=52%3A1816&t=FFaJxIGmxna1hwKT-1) (Figma) from May 2022. Not filed as issues yet.

**Comment by GBKS** [comment]
Thank you, I updated the main comment with your tweaks. No need to re-publish the whole thing though, you can just let me know what to change.

**Comment by mouxdesign** [comment]
Perfect thank you. Will comment here when an issue's been filed with a link to the issue.

**Comment by mouxdesign** [comment]
Adding BTC Pay:

Web shop accessibility testing; Paid Invoice, Pull Payment, Create Invoice
[Issue](https://github.com/btcpayserver/btcpayserver/issues/4986#issuecomment-1554227958)

**Comment by GBKS** [comment]
Closing this issue since it's been inactive for 3+ years. I went through the individual issues listed above and also left comments or closed ones that were stale. Pretty sure that we can consider things abandoned after multiple years of inactivity. Especially if the fixes are trivial to implement, as most accessibility things are. With linters and AI, it should be pretty straightforward to create accessible interfaces.
